source/features/split-issue-pr-search-results.tsx

```diff
@@ -12,20 +12,51 @@ function cleanLinks(): void {
 	}
 }
 
+function getPageSearchQuery(): SearchQuery {
+	const searchParameters = new URLSearchParams(location.search);
+	return new SearchQuery(searchParameters);
```

Comment:
`getPageSearchQuery` be dropped, SearchQuery accepts location
```suggestion
	return new SearchQuery(location);
```

Reply:
This only works if the `searchQuery` isn't modified otherwise `location.search` is also changed.
So it would not work for `createUrl`, didn't personally like that side effect from `SearchQuery` so explicitly created a new copy 😅

Reply:
Indeed, I forgot how it worked. I think the issue here was that we're rebuilding both links from scratch (`createUrl`) starting from `location` instead of just changing the existing `Issues` link. Would you like to try that? This would also avoid the other comment (https://github.com/sindresorhus/refined-github/pull/3837/files#r547508824) since they'll already be in the link

Reply:
Done this :+1: